src/components/access_group/AccessGroupTerms.vue

```diff
@@ -22,7 +22,7 @@
       <h1 class="group-terms__header">
         {{ fluent('access-group_terms', 'heading') }}
       </h1>
-      <p>{{ termsContent }}</p>
+      <p v-html="formattedTerms"></p>
```

Comment:
The original implementation seemed to lack actually presenting the TOS as markdown?